### ISSUE TITLE
Add equality to Calendar object 

### ISSUE THREAD

**Issue by Hugodehu** [builttoroam/device_calendar#491]
**Describe the solution you'd like**
Is it possible to add the equality property to the Calendar object for tests purpose ?

**Comment by thomassth** [comment]
Is there an native API equivalent to this?

**Comment by Hugodehu** [comment]
Yes I think with the native quick action on the class itself allows you to generate the constructor, the equality, the toString method etc...
